Makefile

```
@@ -82,7 +82,7 @@ lint: install-lint-deps ## Lint Go code
 
 ci-lint: install-lint-deps ## On ci only lint newly added Go source files
 	@echo "==> Running linter on newly added Go source files..."
-	GO111MODULE=on golangci-lint run --new-from-rev=origin/master ./...
+	GO111MODULE=on golangci-lint run --new-from-rev=HEAD~1 ./...
```

Ah I went looking on the interwebs and found out about [`git merge-base`](https://git-scm.com/docs/git-merge-base)( [here](https://stackoverflow.com/questions/1527234/finding-a-branch-point-with-git) ):

`git merge-base master HEAD` will give you the first common commit from your HEAD ( or your branch ) to master !!! Which should give the true diff.

```suggestion
	GO111MODULE=on golangci-lint run --new-from-rev=`git merge-base master HEAD` ./...
```

---

Codecov should do the same :|

---

Oh nice, I'll give this a try :tada: